substrate/frame/core-fellowship/src/lib.rs

```diff
@@ -390,14 +395,6 @@ pub mod pallet {
 		}
 
 		/// Approve a member to continue at their rank.
```

Comment:
This looks like an accidental deletion
```suggestion
		/// Approve a member to continue at their rank.
		///
		/// This resets `last_proof` to the current block, thereby delaying any automatic demotion.
		///
		/// `who` must already be tracked by this pallet for this to have an effect.
		///
		/// - `origin`: An origin which satisfies `ApproveOrigin` or root.
		/// - `who`: A member (i.e. of non-zero rank).
		/// - `at_rank`: The rank of member.
```